node/src/components/small_network/config.rs

```diff
@@ -81,6 +82,8 @@ pub struct Config {
     pub public_address: String,
     /// Known address of a node on the network used for joining.
     pub known_addresses: Vec<String>,
+    /// If set, logs all TLS keys to this file.
+    pub keylog_path: Option<String>,
```

Comment:
Suggest using a `PathBuf`

Reply:
I used a string because that's what we have to pass to the OpenSSL API. Is there a benefit for the potentially fallible conversion from `String` (config file) to `PathBuf` (here) just to go back to `String`?